deeposlandia/train.py

```diff
@@ -35,7 +35,8 @@ def add_instance_arguments(parser):
                         help="Aggregate labels with respect to their categories")
     parser.add_argument('-D', '--dataset',
                         required=True,
-                        help="Dataset type (either mapillary, shapes or aerial)")
+                        help=("Dataset type (either mapillary, shapes, aerial"
+                              " or tanzania)"))
```

Comment:
I think we can group some option variables in a same place instead of having 3 different `add_argument` in 3 different modules. The CLI must be centralized in a dedicated module. We can open a new issue for that.

Reply:
I agree to report this to a further issue/PR. Could you write it?

Reply:
done https://github.com/Oslandia/deeposlandia/issues/90

Reply:
That you for creating the issue!